src/include/pixels/RGB-pixel.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "pixel.hpp"
+
+class RGBPixel : public Pixel
+{
+public:
+	RGBPixel(unsigned char r, unsigned char g, unsigned char b);
+
```

Comment:
```suggestion
Add the copy constructor, destructor and operator= with keyword default.
Add "noexcept" to functions which don't throw an exception.
```

Reply:
- I think the `noexcept` specifier could be useful.
- About the explicit marking of constructors and assignment operators as default and deleted... I was not a fan of the idea at first because it looks a little bit ugly to me but it could be useful when writing new code and may be more efficient as it turns out sometimes the move constructors and assignment operators are not implicitly generated by the compiler. So I decided to write it this way at least for the APIs. For the tests... well, we're not looking for great performance there and also we're not using any move constructors there and the classes there are not in some hierarchy so I decided to not specify them. I'm not going to argue if you want to specify them as well.